src/main/java/com/liferay/gw/GradleWrapper.java

```diff
@@ -7,13 +7,19 @@
 	public static void main(String[] args) {
 		GradleExecutor gradleExecutor = new GradleExecutor();
 
+		int exitCode = 0;//normal
+
 		try {
-			gradleExecutor.execute(args);
+			exitCode = gradleExecutor.execute(args);
 		}
 		catch (Exception e) {
 			System.err.println("Error executing gradle wrapper");
 			
 			e.printStackTrace();
+
+			exitCode = -1;//error state?
```

Comment:
Not familiar with the Unix error codes standard, but i think -1 is any generic error